packages/get-platform/src/__tests__/parseOpenSSLVersion.test.ts

```diff
@@ -52,6 +57,11 @@ describe('parseLibSSLVersion', () => {
       content: `/lib/libssl.so.3`,
       expect: '3.0.x',
     },
+    {
+      name: 'libssl 3.1',
+      content: `/lib/libssl.so.3.1`,
+      expect: '3.0.x',
```

Comment:
```suggestion
      expect: '3.0.x',
```
Technically `3.x.x` is better maybe but it doesn't change anything in the logic? (except that we would need to change the built engine names too on the CDN) 
So maybe not necessary to do overall?

Reply:
We're keeping `3.0.x` for now, which we have already tested, and revisit this decision for [Prisma 5](https://www.notion.so/prismaio/Standardise-compilation-target-names-platforms-on-Linux-a4a4c3e91d254f5ca10fcd976ea5a7da).